eng/pipelines/templates/jobs/tests-nightly-python.yml

```diff
@@ -22,9 +22,14 @@ jobs:
       - script: |
           python -m pip freeze
           python -m pip --version
-          pip install setuptools==56.0.0 wheel==0.37.0 tox==3.24.3 tox-monorepo==0.1.2 packaging==21.0
+          pip install setuptools==56.0.0 wheel==0.37.0 tox==3.24.3 tox-monorepo==0.1.2 packaging==21.0 requests
+          pip install $(Build.SourcesDirectory)/tools/azure-sdk-tools[build]
         displayName: Install Dependencies
 
+      - template: /eng/common/testproxy/test-proxy-tool.yml
+        parameters:
+          runProxy: false
+
       - script: |
           python ./scripts/devops_tasks/setup_execute_tests.py "$(BuildTargetingString)" --junitxml="junit/test_results.xml" --toxenv="whl"
```

Comment:
We will disable compatibility by passing filter type `"None"` to this parameter when it is available after another PR merges.